zerocopy-derive/src/lib.rs

```diff
@@ -266,11 +266,48 @@ fn derive_known_layout_inner(ast: &DeriveInput, _top_level: Trait) -> Result<Tok
                 Default::default()
             };
 
+            let field_index =
+                |name| Ident::new(&format!("__Zerocopy_Field_{}", name), ident.span());
```

Comment:
Same comment here and below about the nomenclature "index". If you rename it in the `Field` trait, make sure to similarly rename it here.

Reply:
[Done](https://github.com/google/zerocopy/compare/c7cfc8df03d86f6f93d73c8bc37729ad12eebd96..2cfcc5a8c3baf2a66dbb18fe633ba234a411a155#diff-977c29e6cf079bd7307f811df24f99633df603962785c583145eb6250b0e12daR269-R270)